components/home/integrations/IntegrationCard.module.scss

```diff
@@ -0,0 +1,21 @@
+.cardDetail {
+  border-radius: 5px;
+  box-shadow: 5px 5px 5px 5px lightgrey;
+
+  .integrationCardLink {
+    &:hover {
+      box-shadow: 5px 5px 5px 5px grey;
+    }
+  }
+
+  .pluginName,
+  .badge {
+    color: black;
```

Comment:
We may want to make this change from white font to black font on badges more universal.  With the lighter colors, we'll want things like our blog tags to inherit this change as well.